src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,284 @@
+  </video>
+  <figcaption class="w-figcaption">
+    Images in a grid with their parent element at various aspect ratio dimensions. <a href="https://codepen.io/una/pen/PoGddaw">See demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+## Example: preventing layout shift
+
+Another great feature of `aspect-ratio` is that it can create placeholder space to prevent
+[cumulative layout shift](/cls/) and deliver better [web vitals](/learn-web-vitals/). In this first
+example, loading an asset from an API such as [Unsplash](https://source.unsplash.com/) creates a
+layout shift when the media is finished loading.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted playsinline>
+    <source src="https://storage.googleapis.com/web-dev-assets/aspect-ratio/aspect-ratio-missing.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Video of cumulative layout shift that happens when no aspect ratio is set on a loaded asset. This video is recorded with an emulated 3G network.
+  </figcaption>
+</figure>
+
+Using `aspect-ratio`, on the other hand, creates a placeholder to prevent this layout shift:
+
+```css
+img {
+  width: 100%;
+  aspect-ratio: 8 / 6;
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted playsinline>
+    <source src="https://storage.googleapis.com/web-dev-assets/aspect-ratio/aspect-ratio-set.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Video with a set aspect ratio is set on a loaded asset. This video is recorded with an emulated 3G network. <a href="https://codepen.io/una/pen/GRjLZmG">See demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+### Bonus tip: image attributes for aspect ratio
+
+Another way to set an image's aspect ratio is through [image attributes](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/). If you know the dimensions of the image ahead of time, it is a [best practice](https://web.dev/image-aspect-ratio/#check-the-image's-width-and-height-attributes-in-the-html) to
+set these dimensions as its `width` and `height`.
+
+For our example above, knowing the dimensions are 800px by 600px, the image markup would look like: `<img src="image.jpg"
+alt="..." width="800" height="600">`. If the image sent has the same aspect
+ratio, but not necessarily those exact pixel values, we could still use image
+attribute values to set the ratio, combined with a style of `width: 100%` so
+that the image takes up the proper space. All together that would look like:
+
+```markup
+<!-- Markup -->
+<img src="image.jpg" alt="..." width="8" height="6">
+```
+
+```css
+/* CSS */
+img {
+  width: 100%;
+}
+```
+
+In the end, the effect is the same as setting the `aspect-ratio` on the
+image via CSS, and cumulative layout shift is avoided ([see demo on
+Codepen](https://codepen.io/una/pen/gOwJWoz)).
+
+## Conclusion
+
+With the new `aspect-ratio` CSS property, launching across all modern browsers, maintaining proper
```

Comment:
```suggestion
With the new `aspect-ratio` CSS property, launching across multiple modern browsers, maintaining proper
```
Gonna tweak this to avoid the suggestion that "modern browsers" equals Chromium, Safari, and Firefox specifically.